### ISSUE TITLE
training stops at epoch 6

### ISSUE THREAD

**Issue by charliewang789** [fizyr/keras-retinanet#1436]
After training 6 epoch, it stops and quit without giving any error -

Epoch 00006: saving model to ./snapshots\resnet50_csv_06.h5

what could be wrong?

**Comment by bfialkoff** [comment]
Same here, not necessarily 6, sometime it goes further for me, but same story sudden stop with no explanation.

**Comment by demolakstate** [comment]
Mine stops at epoch 1. increasing patience does not solve the problem. Any help please?
Trace as follows:
Epoch 1/50
2020-09-20 16:43:41.588458: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-09-20 16:43:42.717110: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
   97/10000 [..............................] - ETA: 1:08:14 - loss: 3.3150 - regression_loss: 2.4165 - classification_loss: 0.8985WARNING:tensorflow:Your input ran out of data; interrupting training. Make sure that your dataset or generator can generate at least `steps_per_epoch * epochs` batches (in this case, 500000 batches). You may need to use the repeat() function when building your dataset.
Running network: 100% (15 of 15) |##########################################################################################| Elapsed Time: 0:00:10 Time:  0:00:10
Parsing annotations: 100% (15 of 15) |######################################################################################| Elapsed Time: 0:00:00 Time:  0:00:00
23 instances of class damaged with average precision: 0.1231
69 instances of class undamaged with average precision: 0.6403
mAP: 0.3817

Epoch 00001: saving model to ./snapshots/resnet50_pascal_01.h5
   97/10000 [..............................] - 52s 532ms/step - loss: 3.3150 - regression_loss: 2.4165 - classification_loss: 0.8985
(retinaNet_2) demolakstate@demolakstate:/data/RetinaNet_2/keras-retinanet$
